AISKU/package.json

```diff
@@ -47,7 +47,7 @@
         "grunt-tslint": "^5.0.2",
         "globby": "^11.0.0",
         "magic-string": "^0.25.7",
-        "pako":"^2.0.3",
+        "pako": "^2.0.3",
```

Comment:
the ```npm run setVersion``` script automatically rewrites and formats this file, so that is where this change came from.